### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by janwirth** [bizubee/bizubee#2]
It is unreadable. Removing the text shadow and using a standard grotesque font would greatly profit the project I think.

**Comment by GabrielRatener** [comment]
Yeah, the identifiers are kind of hard to read so I will remove the shadowing for those. I will keep it for keywords to differentiate them from strings for now. What font(s) do you have in mind?

**Comment by janwirth** [comment]
Basically, I think the github stylesheets are just great.
People are used to it and they cover everything I guess.
https://github.com/sindresorhus/github-markdown-css
